android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/profile/PatientProfileViewModel.kt

```diff
@@ -74,11 +75,45 @@ constructor(
               profileViewDataMapper.transformInputToOutputModel(it) as
                 ProfileViewData.PatientProfileViewData
             // TODO only display some overflow menu items when certain conditions are met
+            refreshOverFlowMenu(healthModule = healthModule, patientProfile = it)
           }
       }
     }
   }
 
+  fun refreshOverFlowMenu(healthModule: HealthModule, patientProfile: ProfileData) {
+    if (healthModule == HealthModule.HIV ||
+        healthModule == HealthModule.TRACING ||
+        healthModule == HealthModule.APPOINTMENT
+    ) {
+      val patientMetaFilterType = (patientProfile as ProfileData.DefaultProfileData).filterType
+      when {
+        patientMetaFilterType.equals("exposed-infant", true) ||
```

Comment:
exposed-infant, child-contact, other types to filtering patient here can be check with enum constants,